docs/source/installation/guide_quick/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _installation:guide-quick:
+
+========================
+Quick installation guide
+========================
+
+.. attention::
+
+    AiiDA requires a recent version of Python.
+    Please refer to the `Python Package Index <https://pypi.org/project/aiida-core/>`_ for the minimum required version.
+
+First, install the ``aiida-core`` Python package:
+
+.. code-block:: console
+
+    pip install aiida-core
+
+Next, set up a profile where all data is stored:
+
+.. code-block:: console
+
+    verdi presto
+
+Verify that the installation was successful:
+
+.. code-block:: console
+
+    verdi status
+
```

Comment:
I'd also add here the expected output, so people can directly compare:
```suggestion

.. code-block:: console
 ✔ version:     AiiDA v2.5.1.post0
 ✔ config:      /path/.aiida
 ✔ profile:     presto
 ✔ storage: SqliteDosStorage[/path/.aiida/repository/sqlite_dos_8ee66fde9bd34e7c8fc8e0e29b20c429]: open,
 ⏺ broker:      No broker defined for this profile: certain functionality not available.
 ⏺ daemon:      No broker defined for this profile: daemon is not available.
```

Reply:
The sentence below also technically not true, since the broker and daemon will not show green checkmarks. Maybe showing the output as @GeigerJ2 suggests is enough and we can then remove this sentence?